server/src/main/java/org/spine3/util/Environment.java

```diff
@@ -49,17 +49,18 @@ public static Environment getInstance() {
      * {@code false} otherwise.
      */
     public boolean isAppEngine() {
-        final boolean isVersionPresent = (APP_ENGINE_RUNTIME_VERSION != null) &&
-                !APP_ENGINE_RUNTIME_VERSION.isEmpty();
+        final boolean isVersionPresent = (appEngineRuntimeVersion != null) &&
+                !appEngineRuntimeVersion.isEmpty();
         return isVersionPresent;
     }
 
     /**
      * Returns the current Google AppEngine version
-     * retrieved by the key {@link Environment#APP_ENGINE_RUNTIME_VERSION_KEY}.
+     * retrieved by the key {@link Environment#APP_ENGINE_RUNTIME_VERSION_KEY}
```

Comment:
We do not need to expose this `private static final` constant here. It is not supposed to be visible to public API user.